src/CMakeLists.txt

```diff
@@ -1,18 +1,8 @@
-project (Cuberite)
+target_sources(
+	${CMAKE_PROJECT_NAME} PRIVATE
 
+	Resources/Cuberite.rc
```

Comment:
This is a windows only thing, right?
```suggestion
	$<$<BOOL:WIN32>:Resources/Cuberite.rc>
```

Reply:
Cygwin/MSYS can process them too, and since it's handled automatically by CMake it can be left in.